### PR TITLE
[OSCD] LOLBAS atbroker suspicious creation of ATs

### PR DESCRIPTION
**Pull request by sn0w0tter**
This PR is part of OSCD sprint 2
Rule: sysmon_susp_atbroker_change

Issue: #1014

### CODE REVIEW

File: rules/windows/registry_event/sysmon_susp_atbroker_change.yml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    - attack.persistence
+    - attack.t1547
+logsource:
+    category: registry_event
+    product: windows
+detection:
+    creation:       
+        TargetObject|contains: Software\Microsoft\Windows NT\CurrentVersion\Accessibility\ATs
+    persistance:
+        TargetObject|contains: Software\Microsoft\Windows NT\CurrentVersion\Accessibility\Configuration
```
Comment (e6e6e) [inline rules/windows/registry_event/sysmon_susp_atbroker_change.yml:21]: Hi Mateusz!
Glad to see you among the participants! )
I think it is better to enclose the field values ​​in quotation marks `'`.

Comment (sn0w0tter) [reply rules/windows/registry_event/sysmon_susp_atbroker_change.yml]: yo @e6e6e 
fixed according to suggestion

**Comment by yugoslavskiy** [comment]
Thanks, @sn0w0tter! Good to see you here mate (:

**Review by yugoslavskiy — APPROVED** [review]
@thomaspatzke @Neo23x0 guys, I believe that this PR could be merged (:
please let us know what do you think about it if there is anything you would like to change.

**Pull request merged**